bazel/external/googleurl.patch

```diff
@@ -0,0 +1,116 @@
+diff --git a/base/compiler_specific.h b/base/compiler_specific.h
+index 2962537..6193b56 100644
+--- a/base/compiler_specific.h
++++ b/base/compiler_specific.h
+@@ -7,10 +7,6 @@
+
+ #include "build/build_config.h"
+
+-#if defined(COMPILER_MSVC) && !defined(__clang__)
+-#error "Only clang-cl is supported on Windows, see https://crbug.com/988071"
+-#endif
+-
+ // Annotate a variable indicating it's ok if the variable is not used.
+ // (Typically used to silence a compiler warning when the assignment
+ // is important for some other reason.)
+@@ -212,7 +208,9 @@
+ #endif
+ #endif
+
+-#if defined(__clang__) && __has_attribute(uninitialized)
++#if defined(__clang__)
++#if defined(__has_attribute)
++#if __has_attribute(uninitialized)
```

Comment:
Since we put `/Za` when compiling on Windows, [MSVC's C4067 warning](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4067?view=vs-2019) is treated as an error.

Reply:
I find it interesting given that googletest itself supports msvc, but does not support clang-cl.